cmd/cloud/deployment_objects.go

```diff
@@ -0,0 +1,645 @@
+	return deployment.ConnectionList(airlfowURL, airflowAPIClient, out)
+}
+
+func deploymentConnectionCreate(cmd *cobra.Command, out io.Writer) error {
+	ws, err := coalesceWorkspace()
+	if err != nil {
+		return errors.Wrap(err, "failed to find a valid Workspace")
+	}
+
+	// Silence Usage as we have now validated command input
+	cmd.SilenceUsage = true
+
+	requestedField = requestString
+	// get Airflow URl
+	value, err := inspect.ReturnSpecifiedValue(ws, deploymentName, deploymentID, astroClient, requestedField)
+	if err != nil {
+		return errors.Wrap(err, "failed to find the Deployment Airflow webserver URL")
+	}
+
+	airlfowURL := fmt.Sprintf("%v", value)
+
+	// check connID and connType
+	if connID == "" {
+		return errors.New("a connection ID is needed to create a connection. Please use the '--conn-id' flag to specify a connection ID")
+	}
+
+	if connType == "" {
+		return errors.New("a connection type is needed to create a connection. Please use the '--conn-type' flag to specify a connection type")
+	}
+
+	return deployment.ConnectionCreate(airlfowURL, connID, connType, description, host, login, password, schema, extra, port, airflowAPIClient, out)
+}
+
+func deploymentConnectionUpdate(cmd *cobra.Command, out io.Writer) error {
+	ws, err := coalesceWorkspace()
+	if err != nil {
+		return errors.Wrap(err, "failed to find a valid workspace")
+	}
+
+	// Silence Usage as we have now validated command input
+	cmd.SilenceUsage = true
+
+	requestedField = requestString
+	// get Airflow URl
+	value, err := inspect.ReturnSpecifiedValue(ws, deploymentName, deploymentID, astroClient, requestedField)
+	if err != nil {
+		return errors.Wrap(err, "failed to find the Deployment Airflow webserver URL")
+	}
+
+	airlfowURL := fmt.Sprintf("%v", value)
+
+	// check connID and connType
+	if connID == "" {
+		return errors.New("a connection ID is needed to create a connection. Please use the '--conn-id' flag to specify a connection ID")
+	}
+
+	return deployment.ConnectionUpdate(airlfowURL, connID, connType, description, host, login, password, schema, extra, port, airflowAPIClient, out)
+}
+
+//nolint:dupl
+func deploymentConnectionCopy(cmd *cobra.Command, out io.Writer) error {
+	ws, err := coalesceWorkspace()
+	if err != nil {
+		return errors.Wrap(err, "failed to find a valid Workspace")
+	}
+
+	// Silence Usage as we have now validated command input
+	cmd.SilenceUsage = true
+
+	requestedField = requestString
+	// get To Airflow URls
+	if fromDeploymentName == "" && fromDeploymentID == "" {
+		fmt.Println("Which Deployment should connections be copied from?")
+	}
+	fromValue, err := inspect.ReturnSpecifiedValue(ws, fromDeploymentName, fromDeploymentID, astroClient, requestedField)
+	if err != nil {
+		return errors.Wrap(err, "failed to find the source Deployment Airflow webserver URL")
+	}
+
+	fromAirlfowURL := fmt.Sprintf("%v", fromValue)
+	if toDeploymentName == "" && toDeploymentID == "" {
+		fmt.Println("Which Deployment should receive the connections?")
```

Comment:
what do we do in this case? should we return an error to specify either one and not both or prioritize one over the other